ops-catalog/ops-log-schema.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
you could instead point to the schema in `crates/assemble/src/ops/ops-log-schema.json`, right?

They should be identical (and it would be a problem if they weren't!).

Reply:
I updated `crates/assemble/src/ops` to refer to the schema docs in `ops-catalog` since you mentioned the `ops` module there being effectively deprecated, and also there is an additional schema document for `ops-catalog-stats-schema` in `ops-catalog` so they can all live there instead of being split up.

Interestingly they aren't exactly identical. The schema in `ops-catalog` had a couple of recent-ish updates compared to those in `crates/assemble/src/ops` ([default values](https://github.com/estuary/flow/pull/943/commits/23068a66657a24d5800c703375aac1d4cf0d2261#diff-acf8530839dcae4131d156d15cfeadba04fab44e4abfd07ec96520d33478a437R357), [source property for transforms](https://github.com/estuary/flow/pull/943/commits/23068a66657a24d5800c703375aac1d4cf0d2261#diff-acf8530839dcae4131d156d15cfeadba04fab44e4abfd07ec96520d33478a437R395), [a new log level](https://github.com/estuary/flow/pull/943/commits/23068a66657a24d5800c703375aac1d4cf0d2261#diff-ae74a402356c5dcbade0bdcaa9de6ee9eda7b579c251f9956a8a45957c66c438R8)), and also an interesting [difference in patterns for key/rClock ranges](https://github.com/estuary/flow/pull/943/commits/23068a66657a24d5800c703375aac1d4cf0d2261#diff-acf8530839dcae4131d156d15cfeadba04fab44e4abfd07ec96520d33478a437R491).

I don't know that these differences represent a problem at the moment, and they've been in existence for a while now actually anyway. Let me know what you think.

Reply:
Got it -- definitely use the more recent schemas from ops-catalog then. I believe this hasn't been an issue in practice because we don't actually validate documents being written to these collections from tasks. We only validate them on read, and those schemas are correct.